### DISCUSSION TITLE
Frontend dependencies show vulnerabilities

### DISCUSSION THREAD

**Discussion by lclerigues** [apache/superset#28369] [Q&A / Help]
After having problems initalizing superset (master, pulled on 03 of May)  I tired the solutions showcased in #17076 and had this pop up after the npm install

![image](https://github.com/apache/superset/assets/115978584/183b5b05-6e98-49e5-9b7d-4fd8a10a606c)

audit fix revealed 44 vulnerabilities (4 low, 12 moderate, 27 high, 1 critical) that werent fixable ( 24 high, 1 critical werent even by --force)

the critical being:

- eslint-plugin-translation-vars * : Malware in eslint-plugin-translation-vars

and the high:

- node-fetch  <=2.6.6 : node-fetch forwards secure headers to untrusted sites / The `size` option isn't honored after following a redirect in node-fetch

- xlsx  * : Prototype Pollution in sheetJS / SheetJS Regular Expression Denial of Service (ReDoS)

- d3-color  <3.1.0 : d3-color vulnerable to ReDoS

should I be worried? specially about the malware?

**Comment by lodu** [comment]
`eslint-plugin-translation-vars` is not installed from the NPM registry but its code from the [superset repo](eslint-plugin-translation-vars) itself.
